File: Estrutura_Repetição/C++/Ex1073/Main.cpp
#include <iostream>
#include <math.h>

int main (){

    int valor = 0;
    std::cin >> valor;

    for (int i = 2; i < valor + 1; i += 2) {
        int media = pow(i, 2);
        std::cout << i << "^2 = " << media << std::endl;
    }

    return 0;
}

File: Estrutura_Repetição/C#/Ex1073/Main.cs
using System;

class MainClass {
    public static void Main(string[] args) {
        int valor = Convert.ToInt32(Console.ReadLine());

        for (int i = 2; i < valor + 1; i += 2) {
            int media = (int)Math.Pow(i, 2);
            Console.WriteLine($"{i}^2 = {media}");
        }
    }
}